pep-0440-versioning.rst

```diff
@@ -1084,7 +1103,7 @@ uploaded distributions. Direct references are intended as a tool for
 software integrators rather than publishers.
 
 Depending on the use case, some appropriate targets for a direct URL
-reference may be a valid ``source_url`` entry (see PEP 426), an sdist, or
+reference may be a valid ``source_url`` entry (see PEP 508), an sdist, or
```

Comment:
I think this is actually just a gone concept: direct references can only be sdists or wheels now.